### PR TITLE
Update configure-endpoints-non-windows.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the offboarding section. Please let me know if there are more details that needs to be added.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/6370

File: microsoft-365/security/defender-endpoint/configure-endpoints-non-windows.md [hunk 1]
```diff
@@ -58,13 +58,10 @@ You'll need to take the following steps to onboard non-Windows devices:
 
 ## Offboard non-Windows devices
 
-1. Follow the third party's documentation to disconnect the third-party solution from Microsoft Defender for Endpoint.
+For macOS and Linux devices, you can choose to offboard through Microsoft Defender for Endpoint.
+ 1. In the navigation pane, select ***Settings** \> **Offboard** \> **Select Operating System to start the offboarding Process**
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/configure-endpoints-non-windows.md]:
```suggestion
For macOS and Linux devices, you can choose to offboard through Microsoft Defender for Endpoint. In the navigation pane, select ***Settings** \> **Offboard** \> **Select Operating System to start the offboarding Process**.
```

File: microsoft-365/security/defender-endpoint/configure-endpoints-non-windows.md [hunk 2]
```diff
@@ -58,13 +58,10 @@ You'll need to take the following steps to onboard non-Windows devices:
...
 
-2. Remove permissions for the third-party solution in your Azure AD tenant.
-   1. Sign in to the [Azure portal](https://portal.azure.com).
-   2. Select **Azure Active Directory > Enterprise Applications**.
-   3. Select the application you'd like to offboard.
-   4. Select the **Delete** button.
+You can also Offboard non-windows devices by disabling the third party integration.Enable coverage for devices running non-Windows platforms by [integrating third-party solutions](https://securitycenter.windows.com/interoperability/partners).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/configure-endpoints-non-windows.md]:
```suggestion
You can also offboard non-Windows devices by disabling the third-party integration. Enable coverage for devices running non-Windows platforms by [integrating third-party solutions](https://securitycenter.windows.com/interoperability/partners). 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: microsoft-365/security/defender-endpoint/configure-endpoints-non-windows.md (hunk 2 above)
Comment (Oppedijk) [reply microsoft-365/security/defender-endpoint/configure-endpoints-non-windows.md]: This URL is not publicly accessible, please provide a public URL @JohanFreelancer9

**Comment by yogkumgit** [comment]
@msbemba

File: microsoft-365/security/defender-endpoint/configure-endpoints-non-windows.md (hunk 2 above)
Comment (JohanFreelancer9) [reply microsoft-365/security/defender-endpoint/configure-endpoints-non-windows.md]: @msbemba, could you follow up, please?

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**